### ISSUE TITLE
New Rubygems release

### ISSUE THREAD

**Issue by cseelus** [gregschmit/recurring_select#160]
`recurring_select` from master still works fine with Rails 7, using it in (pre) production.

Thanks to the contributors for all their work.

Let me know if I can help to cut a new release for Rubygems.

**Comment by saturnflyer** [comment]
@gregschmit do you need assistance maintaining this gem?

**Comment by gregschmit** [comment]
I'm sorry I missed this! Yes I can definitely do a release.

To answer your question @saturnflyer: I did recently have my first baby, so I haven't had the time I would like to clean up and improve this project. When I first became maintainer for this project I got the bugs fixed so I could use it in Rails 6.1/7 projects, however I would really like to do more cleanup of the project (like remove coffeescript as dependency), and I know you're the author of a really nice fork of this project, so if you would like to help with this project I'd be more than happy to make you a contributor.
